hunting-buddy.lic

```diff
@@ -253,11 +265,39 @@ class HuntingBuddy
     stop_on_low_skills && stop_on_low_skills.flatten.any? { |skill| DRSkill.getxp(skill) < @stop_on_low_threshold }
   end
 
+  # Is your current encumbrance at or higher than the threshold?
+  # Threshold is a number from 0 (None) to 11 (It's amazing you aren't squashed!)
+  def encumbered?(threshold)
+    return false unless threshold
+    fput('encumbrance') # drinfomon script will parse the output then update DRStats
+    pause 1 # give drinfomon a chance to update DRStats so we use the value from now, not last time
+    encumbrance_index = [
+      "None",
```

Comment:
What do you think about using this instead?

https://github.com/rpherbig/dr-scripts/blob/19d06c8de3ece9bd2185960706a39c07df822dbd/common.lic#L52

Reply:
Ha! I didn't know that was there, thanks